server/src/utils/ServerState.ts

```diff
@@ -5,15 +5,14 @@ let serverReady = true;
 
 const setReady = (ready: boolean) => {
   LOGGER.log(`Updating server state to ${ready ? 'release' : 'hold'} traffic`);
-
   serverReady = ready;
 };
 
 const isReady = () => serverReady;
 
 const setGlobalTournamentValid = async (prisma: PrismaClient, valid: boolean) => {
   await prisma.serverState.upsert({
-    where: { id: 1 },
+    where: { id: '' },
```

Comment:
I don't think this will work.  
The upsert needs to be replaced with a findFirst, and an insert if needed